packages/shared/src/utils/keys.ts

```diff
@@ -55,6 +55,7 @@ export function isLegacyFrontendApiKey(key: string) {
 }
 
 export function createDevOrStagingUrlCache() {
+  // TODO: Check if we can merge it with `./instance.ts#isStaging()`
   const DEV_OR_STAGING_SUFFIXES = [
```

Comment:
I will cater this in another iteration where we will evaluate the exports of the `@clerk/shared`